### PR TITLE
Update symptom report buttons

### PR DESCRIPTION
**Pull request by mellowgeek**

### CODE REVIEW

**Comment by mellowgeek** [comment]
@i-schuetz made the suggested changes to the symptom report first, only issue is that there isn't any spacing between each item in the table view. Any suggestions? Other feedback also welcome.

**Comment by i-schuetz** [comment]
@mellowgeek you should be able to solve that by adding top/bottom constraints with space to the button in the cells.

File: CoEpi/ui/styles/ButtonStyles.swift [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+struct ButtonStyles{
+    
+    private init(){}
+    
+    static func applyStyleHomeCard(to: UIButton){
+        to.layer.cornerRadius = 15
+        applyStyleShadows(to: to)
+    }
+    
+    static func applyStyleSelectedButton(to:UIButton){
```
Comment (i-schuetz) [inline CoEpi/ui/styles/ButtonStyles.swift:13]: Use local parameter name: `(to button: UIButon)`. `to:` is only for calling it.

File: CoEpi/ui/styles/ButtonStyles.swift [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import Foundation
+import UIKit
+
+struct ButtonStyles{
```
Comment (i-schuetz) [inline CoEpi/ui/styles/ButtonStyles.swift:4]: This LGTM. It's not the fanciest styling system, but it works, until we start using SwiftUI and figure out best practices there.

**Pull request merged**